src/Core/src/ImageSources/IImageSourceServiceConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Maui
+{
+	public interface IImageSourceServiceConfiguration : IDictionary<string, string>
```

Comment:
This is all a hacky way so that we can randomly set configuration from the XAML side. This is right now used for the ImageDirectory for Windows, but either needs to be done better, ow we make this a better concept. OR find the correct way to do this.